easy-ui-react/src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,83 @@
+@use "../styles/common" as *;
+
+// prettier-ignore
+.statusPromotional {
+  @include component-token("notification", "color", theme-token("color.text.inverse"));
+  @include component-token("notification", "color.background", theme-token("color.background.primary"));
+}
+
+// prettier-ignore
+.statusSuccess {
+  @include component-token("notification", "color", theme-token("color.text.success.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.success.subtle"));
+}
+
+// prettier-ignore
+.statusNeutral {
+  @include component-token("notification", "color", theme-token("color.text.secondary.bold"));
+  @include component-token("notification", "color.background",theme-token("color.background.secondary.subtle"));
+}
+
+// prettier-ignore
+.statusWarning {
+  @include component-token("notification", "color", theme-token("color.text.inverse.inverse.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.warning.subtle"));
+}
+
+// prettier-ignore
+.statusError {
+  @include component-token("notification", "color", theme-token("color.text.danger.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.danger.subtle"));
+}
+
+.typeToast {
+  display: inline-flex;
+}
+
+.typeAlert {
+  display: flex;
+  width: 100%;
+}
+
+// prettier-ignore
+.Notification {
+  @include component-token("notification", "border_radius", design-token("shape.border_radius.base"));
+  @include component-token("notification", "max-width", 1200px);
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  padding: design-token("space.1");
+  border-radius: 0 0 component-token("notification", "border_radius") component-token("notification", "border_radius");
+  max-width: component-token("notification", "max-width");
+  color: component-token("notification", "color");
+  background-color: component-token("notification", "color.background");
+}
```

Comment:
**issue:** i notice that when a toast is open on the story that the `container` and `region` are blocking the cursor from hitting elements underneath it. probably we want `pointer-events: none;` on the container and `pointer-events: auto;` on the `.Notification`

when the toast is open below the tops of some of the buttons can't be clicked due to the container elements cutting off the cursor:

<img width="923" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/ae6b6bc8-6ba0-4027-8d27-8fbe2a91ac14">